acra/configuring-maintaining/general-configuration/acra-poisonrecordmaker.md

```diff
@@ -81,4 +81,30 @@ weight: 11
 * `--vault_tls_transport_enable={true|false}`
 
   Use TLS to encrypt transport with HashiCorp Vault.
-  Default is `false`.
\ No newline at end of file
+  Default is `false`.
+
+## Usage
+
+In general, this tool is used like this:
+
+1) Generate "poison keys" using one of
+   * `acra-keys generate --keystore=vX --poison_record_keys`
+   * `acra-keymaker --keystore=vX --generate_poisonrecord_keys`
+
+2) Use this tool to generate the poison record itself:
+
+   `acra-poisonrecordmaker > poison_record`
```

Comment:
will be great to show example of output and add a note that client's should store poison record in decoded binary format, not in bas64
plus will be nice to add an example with poison record using AcraBlocks with the additional parameter

upd. oh, I'm late with comment